itwin-oauth-demo/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
```

Comment:
@ggetz I was thinking it might be good to keep something like this around and maybe actually solidify the popup login mechanic for development sandcastles. If we want to experiment with other iTwins it's probably quickest to add our user accounts to them instead of a service account meaning we need to do user oauth or pull the token directly from the docs or the auth API request.

Reply:
I've had to go back to using this workflow for user auth for the Reality data and it will definitely be handy to have around. That said I only really added it to this repo to make it easier to share with those reviewing this PR. I'll pull it all out into another project that I can share as needed.